vocode/streaming/synthesizer/eleven_labs_synthesizer.py

```diff
@@ -47,16 +49,17 @@ async def create_speech(
             voice.settings = self.elevenlabs.VoiceSettings(
                 stability=self.stability, similarity_boost=self.similarity_boost
             )
-        url = ELEVEN_LABS_BASE_URL + f"text-to-speech/{self.voice_id}"
+        url = ELEVEN_LABS_BASE_URL + f"text-to-speech/{self.voice_id}/stream"
+        if self.optimize_streaming_latency:
+            assert 0 <= self.optimize_streaming_latency <= 4 
+            url += f"?optimize_streaming_latency={self.optimize_streaming_latency}"
         headers = {"xi-api-key": self.api_key}
         body = {
+            # change the model_id to "eleven_multilingual_v1" for multi-language support 
+            "model_id": self.model_id,
```

Comment:
i think we should follow the earlier pattern for `optimize_streaming_latency` here - only add `model_id` to the body if it's passed in